FrugalApp/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "frugalapp",
+  "main": "server.js",
```

Comment:
We don't need a second server or a second way to start the server, so you can delete this line